### ISSUE TITLE
Radio buttons & Notes

### ISSUE THREAD

**Issue by lbrunsgaard** [stefangabos/Zebra_Form#38]
After latest update by jasondegn (5/23/17) to zebraform.php - the template files displaying forms aren't lining up correctly, especially with radio buttons and notes are not lining up as they did prior to this update.
Any assistance is appreciated...
Thank you!
![zebra_form---note was under text box---2017 05 24](https://cloud.githubusercontent.com/assets/28934637/26422714/99decf44-4090-11e7-9700-2505dfe9bba9.JPG)
![zebra_form---radio buttons2---2017 05 24](https://cloud.githubusercontent.com/assets/28934637/26422715/99e10ab6-4090-11e7-9142-28cd1b5c9a98.JPG)
![zebra_form---radio buttons---2017 05 24](https://cloud.githubusercontent.com/assets/28934637/26422713/99de560e-4090-11e7-8685-a0580fa71039.JPG)

**Comment by lbrunsgaard** [comment]
This is after PHP upgrade and updgraded Zebra_Form.php (2.9.8).

**Comment by lbrunsgaard** [comment]
This started happening after PHP upgrade to 7.1.5 and Zebra Form update to 2.9.8 and this week's zebraform.php (from jasondegn).

**Comment by lbrunsgaard** [comment]
Issue is resolved after tweaking/modifying zebraform.css.

**Comment by stefangabos** [comment]
I'll look into it, just didn't have yet the time. I am using this with Bootstrap for years now and forgot to check how the changes affects the standalone templates...

**Comment by lbrunsgaard** [comment]
I am attaching a copy of our custom zebraform.css for you to compare....Besides formatting issues the calendar/datepicker wouldn't popup either.  
[zebra_form.css.txt](https://github.com/stefangabos/Zebra_Form/files/1135629/zebra_form.css.txt)

**Comment by lbrunsgaard** [comment]
It had to do with div class="clear" for Carriage returns and formatting...

**Comment by njwgh** [comment]
This css issue has been there for a long time

The css that comes with the zebra form install is not the  same as the css used for the examples
and it is not the same as the css used on the Zebra form demo pages
notably

.Zebra_Form div.clear {
    clear: both
}
is missing so cells don't wrap to next line for radio buttons

**Comment by lbrunsgaard** [comment]
Makes sense....Plus we had a popup issue with datepicker.  Just FYI - I attached for your review/analysis/reference.
Have a wonderful day!
